git-pre-commit error preventing commits

I've forked and cloned the repo, but committing anything is throwing the following error. This only happens after running 
`npm install` or `npm update`. It seems to be related to the npm package `precise-commits` https://www.npmjs.com/package/precise-commits?activeTab=readme


```
git commit -m "test"  
△  run hook: pre-commit -> git-pre-commit
gi
> @datagraphics/baker@0.44.0 git-pre-commit /Users/mv/Code/web-components/baker
> precise-commits

t/Users/mv/Code/web-components/baker/node_modules/precise-commits/lib/precise-formatters/prettier.js:18
prettier_1.getSupportInfo().languages.forEach(function (language) {
                                     ^

TypeError: Cannot read property 'forEach' of undefined
    at Object.<anonymous> (/Users/mv/Code/web-components/baker/node_modules/precise-commits/lib/precise-formatters/prettier.js:18:38)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/Users/mv/Code/web-components/baker/node_modules/precise-commits/lib/index.js:15:18)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @datagraphics/baker@0.44.0 git-pre-commit: `precise-commits`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @datagraphics/baker@0.44.0 git-pre-commit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -97,10 +97,10 @@
     "@datagraphics/prettier-config": "^2.0.0",
     "@types/node": "^18.15.11",
     "@vercel/git-hooks": "^1.0.0",
-    "precise-commits": "^1.0.2",
     "np": "^7.6.3",
     "preact": "^10.11.3",
-    "prettier": "^3.0.3"
+    "precise-commits": "^1.0.2",
+    "prettier": "^2.8.8"
   },
   "prettier": "@datagraphics/prettier-config"
 }

